Sources/LibAuk/Storage/SecureStorage.swift

```diff
@@ -72,20 +73,20 @@ class SecureStorage: SecureStorageProtocol {
             Keys.mnemonic(seed.data)
         }
         .tryMap { [unowned self] in
-            try self.saveKeyInfo(mnemonic: $0)
+            try self.saveKeyInfo(mnemonic: $0, passphrase: password ?? "")
         }
         .eraseToAnyPublisher()
     }
     
-    func importKey(words: [String], name: String, creationDate: Date?) -> AnyPublisher<Void, Error> {
+    func importKey(words: [String], password: String?, name: String, creationDate: Date?) -> AnyPublisher<Void, Error> {
```

Comment:
Let's make this function argument overloading by having the password default value to be empty. Also, let's call it `passphrase` instead of `password`.